app/src/main/res/drawable/button_default_background.xml

```diff
@@ -2,5 +2,6 @@
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
     <item android:drawable="@drawable/button_selected_background" android:state_pressed="true" />
     <item android:drawable="@drawable/button_highlighted_background" android:state_selected="true"/>
-    <item android:drawable="@drawable/button_background"/>
+    <item android:drawable="@drawable/button_background" android:state_enabled="true"/>
```

Comment:
I don't think you have to specify `android:state_enabled="true"`. You can just move your enabled drawable to the bottom of the list after the disabled file. These drawable selectors act like if-blocks, so if it doesn't find a matching state, it'll just default to the last on listed.
So swap lines 5 and 6 and remove the state from the new line 6 -
`<item android:drawable="@drawable/button_background"/>`

Reply:
The same goes for the other two files below.